docs/apache-airflow/howto/operator/python.rst

```diff
@@ -90,7 +90,57 @@ If additional parameters for package installation are needed pass them in ``requ
 All supported options are listed in the `requirements file format <https://pip.pypa.io/en/stable/reference/requirements-file-format/#supported-options>`_.
 
 
+.. _howto/operator:ShortCircuitOperator:
+
+ShortCircuitOperator
+========================
+
+Use the :class:`~airflow.operators.python.ShortCircuitOperator` to control whether a pipeline continues
+if a condition is satisfied or a Truthy value is obtained. The evaluation of this condition and Truthy value
+is done via the output of a ``python_callable``. If the ``python_callable`` returns True or a Truthy value,
```

Comment:
Why are *Truthy* and *Falsy* capitalised? The existing code base does not do this and simply writes them in lower-cased form.

Reply:
Honestly, I didn't check the code base. My thought was Truthy and Falsy were analogous to True and False which are capitalized. Happy to keep this consistent with the code base.